entry/entry_test.go

```diff
@@ -133,9 +133,12 @@ func TestRead(t *testing.T) {
 }
 
 func TestCopy(t *testing.T) {
+	now := time.Now()
```

Comment:
To be consistent:
```suggestion
	now := timeNow()
```

Reply:
I disagree on this one. `timeNow` only exists for when a timestamp cannot be otherwise controlled. 

Specifically, `entry.New` must set `ObservedTimestamp`. In order to test that this is done, we need to know ahead of time what that timestamp should be. So, `entry.New` calls `timeNow`, which is `time.Now` in production, but can be overridden for specific test cases where the timestamp cannot otherwise be controlled.